### PR TITLE
Set onclick for profile

### PR DESCRIPTION
**Pull request by achowdhury3762**

### CODE REVIEW

**Comment by achowdhury3762** [comment]
Not sure if EmployeeView was supposed to be specific to every individual

File: app/src/main/java/nyc/c4q/grocerystore/employeeHandbook/employees/Hyunjoo.java [hunk 1]
```diff
@@ -17,6 +17,6 @@
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.hyunjoo,container,false);
+            return inflater.inflate(R.layout.hyunjoo,container,false);
```
Comment (jsjrobotics) [inline app/src/main/java/nyc/c4q/grocerystore/employeeHandbook/employees/Hyunjoo.java]: @achowdhury3762 shouldn't be modifying other peoples files

**Review by jsjrobotics — COMMENTED** [review]
This looks good and you are the first to complete the task. Remove the modification from hyunjoo so she doesn't get conflicts and I will merge

File: app/src/main/java/nyc/c4q/grocerystore/employeeHandbook/employees/Hyunjoo.java (hunk 1 above)
Comment (achowdhury3762) [reply app/src/main/java/nyc/c4q/grocerystore/employeeHandbook/employees/Hyunjoo.java]: 👍

**Comment by jsjrobotics** [comment]
Looks good. merged

**Pull request merged**